modules/standard/BigInteger.chpl

```diff
@@ -3434,15 +3434,25 @@ module BigInteger {
     }
   }
 
-
+  /*
+   An `InversionError` is thrown if a `bigint.invert()` is attempted with
+   invalid arguments that result in a non-existant inverse. Specifically,
+   if the arguments cause a divide by zero, this error notifies the caller
+   that the internal value of the `bigint` was left in an undefined state.
+   */
   class InversionError : Error {
+
+    /* Create an :class:`InversionError` with the default error message: `Inverse does not exist` */
     proc init() {
-      super.init("inverse does not exist");
+      super.init("Inverse does not exist");
     }
   }
 
-  // invert
-  proc bigint.invert(const ref a: bigint, const ref b: bigint) : int throws {
+  /* A parameter to select between the new and depracated overlaods of `bigint.invert()` */
+  config param INVERT_RETURN_INT = true;
```

Comment:
I think that's fine, though I'd maybe use camelCase instead of ALL_CAPS_WITH_UNDERSCORE for the name, I think it matches other configs in the libraries better if I remember right.

It doesn't matter too much - I'd expect this flag to be fairly short-lived, since it won't be necessary after the deprecated version of invert has been removed.